app/src/main/java/com/mobileproto/hireddit/hireddit/visuals/SpeechFragment.java

```diff
@@ -76,6 +78,12 @@ public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         if (getArguments() != null) {
         }
+        ShakeDetector.create(this.getContext(), new ShakeDetector.OnShakeListener() {
+            @Override
+            public void OnShake() {
+                shake();
+            }
+        });
```

Comment:
@nmohamed Do you have any tips regarding making the shake image? I just know you're a pro at this sort of thing.